nautobot/extras/plugins/urls.py

```diff
@@ -11,6 +11,7 @@
 apps_patterns = [
     path("installed-apps/", views.InstalledAppsView.as_view(), name="apps_list"),
     path("installed-apps/<str:app>/", views.InstalledAppDetailView.as_view(), name="app_detail"),
+    path("marketplace/", views.MarketplaceView.as_view(), name="marketplace"),
```

Comment:
Maybe `name="apps_marketplace"` to be consistent?

Reply:
not a problem, but just to make sure - would you like me to rename everything "marketplace" to "apps_marketplace" or just the two places you mentioned?

Reply:
I'm fine with leaving the url pattern as just `marketplace/`. Those were the two places that jumped out at me, not sure if there are others of significance.

Reply:
fee054d2c1377591d9994d3eb0efbde9c83f53a5